pkg/util/file_indexer.go

```diff
@@ -276,3 +266,13 @@ func write(filePath string, fi *FileIndex) error {
 	// 0600 is the mask used when a file is created using os.Create hence defaulting
 	return ioutil.WriteFile(filePath, jsonData, 0600)
 }
+
+// writes the map of walked files and info about them, in a file
```

Comment:
Writing the comment like `WriteFile writes the map...` the code style would be followed, no?